Sources/DateExtensions.swift

```diff
@@ -32,10 +32,26 @@ extension Date {
             self = rfc850
             return
         }
-        if let asctime =  Date(fromString: httpDateString, format: "EEE MMM d HH':'mm':'ss yyyy") {
+        if let asctime = Date(fromString: httpDateString, format: "EEE MMM d HH':'mm':'ss yyyy") {
             self = asctime
             return
         }
+        if let iso8601DateOnly = Date(fromString: httpDateString, format: "yyyy-MM-dd") {
+            self = iso8601DateOnly
+            return
+        }
+        if let iso8601DateHrMinOnly = Date(fromString: httpDateString, format: "yyyy-MM-dd'T'HH:mmZZZZ") {
```

Comment:
My bad it should be a single 'Z'. I'm surprised that it still converts correctly